src/sentry/runner/initializer.py

```diff
@@ -283,6 +274,17 @@ def configure_structlog():
 def initialize_app(config, skip_service_validation=False):
     settings = config["settings"]
 
+    # Just reuse the integration app for Single Org / Self-Hosted as
+    # it doesn't make much sense to use 2 separate apps for SSO and
+    # integration.
+    if settings.SENTRY_SINGLE_ORGANIZATION:
+        options_mapper.update(
+            {
+                "github-app.client-id": "GITHUB_APP_ID",
+                "github-app.client-secret": "GITHUB_API_SECRET",
+            }
+        )
```

Comment:
This change is to align it with this file in general. Can move it to a separate PR if you think that's better.

Reply:
I would prefer this in a separate PR yeah